src/renderers/webxr/WebXRManager.js

```diff
@@ -191,6 +192,24 @@ class WebXRManager extends EventDispatcher {
 
 		};
 
+		this.getOffsetReferenceSpace = function () {
+
+			return offsetReferenceSpace;
+
+		};
```

Comment:
Seems like the example is not using this one. When would it be needed?

Reply:
Not sure tbh but I imagine it could be useful. Offset reference space tells you where is the center of the VR playing area in threejs world. Some experiences might want to put other non-player objects relative to that coordinate frame and its handy to get that matrix from WebXRManager. 

On the other hand, application can also keep track of that matrix.

Up to you. lmk what you think.

Reply:
I see... Yeah, seems like this one is handy.

Reply:
@mrdoob Now I'm thinking maybe we should amend `getReferenceSpace()` instead:
```javascript
		this.getReferenceSpace = function () {

			return offsetReferenceSpace || referenceSpace;

		};
```

Reply:
That makes sense...

How about renaming `offsetReferenceSpace` to `customReferenceSpace` too?

```js
		this.getReferenceSpace = function () {

			return customReferenceSpace || referenceSpace;

		};

		this.setReferenceSpace = function ( value ) {

			customReferenceSpace = value;

		};
```

Reply:
Done! Merge?